docs/configurations.cn.md

```diff
@@ -223,7 +223,11 @@ Following configs control indexing behavior, which tags incoming records as eith
 
 [withIndexConfig](#withIndexConfig) (HoodieIndexConfig) <br/>
 <span style="color:grey">This is pluggable to have a external index (HBase) or use the default bloom filter stored in the Parquet files</span>
-        
+
+##### withIndexClass(indexClass = "x.y.z.UserDefinedIndex") {#withIndexClass}
+Property: `hoodie.index.class` <br/>
+<span style="color:grey">Full path of user-defined index class and must extends HoodieIndex class. It will take precedence over the `hoodie.index.type` configuration if specified</span>
```

Comment:
must extend?

Reply:
> xtend?

In fact, it is must be a subclass of HoodieIndex, will fix the descrption. :)

Reply:
> must extend?

In fact, it is must be a subclass of HoodieIndex, will fix the descrption. :)